docs/howto/object-storage/s3/object-lock.md

```diff
@@ -0,0 +1,284 @@
+# Object lock
+
+Object Lock is a feature in S3 that enables you to lock your objects
+to prevent them from being deleted or overwritten. This feature provides
+a storage solution based on the **Write Once, Read Many (WORM)** model,
+which can be helpful when you want to maintain data integrity,
+comply with regulations, or retain data for a specific period.
+
+> Are you looking to set the expiration/retention time for your
+> objects? Then you should check our [Object expiry](expiry.md) guide.
+
+## Object Lock modes
+
+Object Lock offers two methods for managing object retention:
+**Retention periods** and **Legal hold**.
+
+### Retention periods
+
+Retention periods specify the length of time that an object
+remains locked and protected from being overwritten or deleted.
+When you set a retention period, the object is safeguarded for
+the specified time period. You can set the retention period in
+either days or years, with a **minimum of one day** and
+no maximum limit.
+
+In addition to retention periods, you can choose the retention
+mode that applies to your objects. At the moment only the
+**Compliance** mode is available.
+
+#### Compliance mode
+
+> Provides the highest level of protection,
+> and is the best option when storing compliant data.
+
+Compliance mode is recommended when storing compliant data,
+as it prevents objects from being overwritten or deleted
+by any user. If you configure an object with this mode, you
+cannot shorten or change its retention period, ensuring that
+the data remains **secure** and **compliant** with regulatory
+requirements.
+
+> Should be used when you want to ensure that no user, including
+> the administrator user, can delete objects during a defined
+> retention period.
+
```

Comment:
This section was previously a side-by-side comparison between Governance and Compliance mode, where it made sense. Now that the bit about Governance mode has been thrown out, it feels oddly disconnected.

Can we edit it down to this?

```suggestion
Compliance mode is recommended when storing compliant data,
as it prevents objects from being overwritten or deleted
by any user. If you configure an object with this mode, you
cannot shorten or change its retention period, ensuring that
the data remains **secure** and **compliant** with regulatory
requirements.
```